easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.7-iomkl-2018a-R-3.5.0.eb

```diff
@@ -0,0 +1,937 @@
+easyblock = 'Bundle'
+
+name = 'R-bundle-Bioconductor'
+version = '3.7'
+biocver = version
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'iomkl', 'version': '2018a'}
+
+dependencies = [
+    ('R', '3.5.0', '-X11-20180131'),
+    ('HDF5', '1.10.1'),  # for rhdf5
+    ('MariaDB-connector-c', '2.3.5'),  # for RMySQL
+]
+
+exts_default_options = {
+    'source_urls': [
+        'https://bioconductor.org/packages/3.7/bioc/src/contrib/',
```

Comment:
@edmondac download is failing for `Rsamtools`, probably because the version was updated

This can be mitigated by updating the `Rsamtools` version (current is `1.32.2`), but also by including the archive source URL, cfr. https://github.com/easybuilders/easybuild-easyconfigs/pull/6311/files

Reply:
I've added the archive line